src/Perpetuum/Modules/BlobEmissionModulatorModule.cs

```diff
@@ -61,10 +61,19 @@ protected override void OnAction()
             if (zone == null)
                 return;
 
-            var terrainLock = GetLock().ThrowIfNotType<TerrainLock>(ErrorCodes.InvalidLockType);
+            Position targetPosition = new Position();
```

Comment:
Addressed in new changes - 
Position is a non-nullable class.  So we do the next best thing and check if it is 0,0,0 or the same object hash.

Reply:
Is 0, 0, 0 *always* an invalid position on *every* map?
Personally, I don't believe the check again 0, 0, 0 is necessary anymore because of the exclusive if statements (unless there's an extra type of lock?)

My personal feeling on this is that the struct should be marked nullable `Position? position = null` or, should be unreachable if a valid position isn't given by the selective `if` conditions.